types/json/binary_functions.go

```diff
@@ -1008,6 +1008,9 @@ func PeekBytesAsJSON(b []byte) (n int, err error) {
 	case TypeCodeLiteral:
 		n = valTypeSize + 1
 		return
+	case TypeCodeOpaque:
```

Comment:
Are there any tests covered

Reply:
I added a test in the codec, and it's covered now :beers: .